src/algorithm/SORT.hpp

```diff
@@ -0,0 +1,62 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// SORT kernel reference implementation:
+///
+/// std::sort(x+ibegin, x+iend);
+///
+
+#ifndef RAJAPerf_Algorithm_SORT_HPP
+#define RAJAPerf_Algorithm_SORT_HPP
+
+#define SORT_DATA_SETUP \
+  Real_ptr x = m_x;
+
+#define SORT_STD_ARGS  \
```

Comment:
These two macros are the same. Why have both?

Reply:
removed one